sklearn/utils/estimator_checks.py

```diff
@@ -1136,8 +1128,6 @@ def check_classifiers_train(name, classifier_orig):
             assert_array_equal(np.argmax(y_prob, axis=1), y_pred)
             # check that probas for all classes sum to one
             assert_allclose(np.sum(y_prob, axis=1), np.ones(n_samples))
-            # raises error on malformed input
-            assert_raises(ValueError, classifier.predict_proba, X.T)
```

Comment:
can you tell me why this change is necessary?

Reply:
because it's a duplicate of the next two lines